palindrome_finder.py

```
@@ -6,14 +6,19 @@ def is_palindrome(word):
     return word == word[::-1]
 
 
+def clean_word(word):
+    # Remove punctuation and numbers
+    return "".join(c for c in word if c.isalpha())
+
+
 def find_palindromes(filename):
```

Ruff (0.8.2)

Missing return type annotation for public function `find_palindromes`

Add return type annotation: `None`

(ANN201)